source/_integrations/griddy.markdown

```diff
@@ -1,8 +1,8 @@
 ---
 title: Griddy Power
-description: Instructions on how to integrate griddy prices into Home Assistant.
+description: Instructions on how to integrate griddy real-time electricity prices into Home Assistant.
 ha_category:
-  - Sensor
+  - Energy
```

Comment:
I think it belongs in both. It provides sensors...

Reply:
Cool